lib/l10n_utils/translation.py

```diff
@@ -6,55 +6,51 @@
 # and we don't need anything nearly as complex.
 
 from django.conf import settings
-
-from _threading_local import local
-
-_active = local()
+from django.utils import translation
 
 
 def activate(language):
-    """
-    Installs the given language as the language for the current thread.
-    """
-    _active.value = language
+    return translation.activate(language)
 
 
 def deactivate():
     """
     Uninstalls the currently active language so that further _ calls
     will resolve against the default language, again.
     """
-    if hasattr(_active, "value"):
-        del _active.value
+    return translation.deactivate()
 
 
-def _fix_case(locale):
-    """Convert lowercase locales to uppercase: en-us -> en-US"""
-    parts = locale.split("-")
-    if len(parts) == 1:
-        return locale
-    else:
-        return f"{parts[0]}-{parts[1].upper()}"
+def _fix_case(lang_code):
```

Comment:
**note:** This could maybe be merged with `bedrock.base.i18n.normalize_langauge` (which does more than this function) as long as we add a flag/option to limit behaviour to be the same as _fix_case.

Reply:
I'm in support of that. The less code we have that does similar things the easier to understand.